package.json

```diff
@@ -25,30 +25,37 @@
     "Richard Gibson <richard.gibson@gmail.com>"
   ],
   "files": [
-    "index.js"
+    "index.js",
+    "types/index.d.ts"
   ],
+  "types": "types/index.d.ts",
   "dependencies": {
+    "unist-util-is": "github:syntax-tree/unist-util-is#a23423",
```

Comment:
This is a work-around while latest typings for unist-util-is are not yet on NPM.
:hand: :stop_sign: Hold off on merging til it is available on NPM.

Reply:
Thanks for releasing `unist-util-is` @wooorm :bow:

The dependency has been updated to stable version 4 release here.